Doc/library/importlib.rst

```diff
@@ -1048,7 +1048,11 @@ find and load modules.
 
 .. class:: ModuleSpec(name, loader, *, origin=None, loader_state=None, is_package=None)
 
-   A specification for a module's import-system-related state.
+   A specification for a module's import-system-related state.  This is
+   typically exposed as the module's ``__spec__`` attribute.  In the
+   descriptions below, the parenthetical "dunder" version names the equivalent
+   attribute available directly on the module object.
+   E.g. ``module.__spec__.name == module.__name__``.
```

Comment:
This isn't quite true, since some of the names are different.

PEP 451 has the equivalence table, but we should probably move that into the library reference somewhere: https://www.python.org/dev/peps/pep-0451/#attributes

(Perhaps here?)

Reply:
Then what exactly *are* those parenthetical names supposed to describe?

Reply:
Maybe @brettcannon can shed some light on what those parenthetical names are supposed to mean?

Reply:
https://docs.python.org/3/library/importlib.html#importlib.machinery.ModuleSpec has the name equivalence in parentheses from dunder name to spec name.

Reply:
@brettcannon I think when you say "the name equivalence in parentheses from dunder name to spec name", you're agreeing with what I wrote in the doc change.  But then I'm confused about what @ncoghlan is saying that "some of the names are different".  It's true that the names in the spec are not the same as the dunder names on the module object, but that's not what I'm saying in the new docs.  At least I'm *trying* to say what @brettcannon is saying. :)   Is my wording too confusing?

Reply:
I think using the `dunder` jargon is what confused me, when combined with the use of an example where they *are* the same. (The fact I was looking at the change in isolation without any of the attribute descriptions in front of me didn't help).

I'd find this clearer:

> In the descriptions below, the name given in parentheses is the corresponding attribute available directly on the module object. These values are equivalent in most cases (e.g. ``module.__file__ == module.__spec__.origin``), but may differ if the module attribute has been modified at runtime (e.g. when overriding ``__name__`` or updating ``__path__``).

Reply:
Thanks @ncoghlan that was helpful clarification.  I pushed an update to this section.